.cbmc-batch/jobs/aws_array_list_comparator_string/aws_array_list_comparator_string_harness.c

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ * this file except in compliance with the License. A copy of the License is
+ * located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <aws/common/string.h>
+#include <proof_helpers/make_common_data_structures.h>
+#include <proof_helpers/proof_allocators.h>
+#include <proof_helpers/utils.h>
+#include <stddef.h>
+
+const size_t MAX_STRING_LEN = 16;
```

Comment:
You're defining identical values in each of the harnesses.

I'm not going to block this PR, but please consider defining this once in a tiny makefile, and including that makefile from each of the harness-specific makefiles. Then write a comment above the define explaining why you chose the value 16. That way we have a way to change the value from one place if necessary, while retaining the ability to override it on a per-harness basis.

Reply:
Done